vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -139,6 +139,8 @@ void UwpReactInstance::Start(const std::shared_ptr<IReactInstance> &spThis, cons
     devSettings->useJITCompilation = settings.EnableJITCompilation;
     devSettings->debugHost = settings.DebugHost;
     devSettings->debuggerPort = settings.DebuggerPort;
+    devSettings->sourceBundleHost = settings.SourceBundleHost;
```

Comment:
>devSettings [](start = 4, length = 11)

The UwpReactInstance class is not used anymore. Please make sure that you change the ReactInstanceWin instead.

Reply:
The DevSettings does not have such fields. There is no compilation error because the file is not compiled.

---
In reply to: [428960875](https://github.com/microsoft/react-native-windows/pull/4979#discussion_r428960875) [](ancestors = 428960875)

Reply:
Let's delete this file?

Reply:
Yes, we must start deleting such files to make our code smaller. I can start deleting the unused files.

Reply:
My bad, thanks for catching this!
Noob question, how can I test an app with ReactInstanceWin?
I tried using `packages/microsoft-reactnative-sampleapps`, but that app allowed me to edit hostname/port without this change.

Reply:
I usually use the packages/microsoft-reactnative-sampleapps. We are building a new set of unit tests. In future we will automate testing of all such features.